<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogToInsert</class>
 <widget class="QDialog" name="DialogToInsert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="11" column="3">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="15" column="3">
    <widget class="QLineEdit" name="lineEdit_9"/>
   </item>
   <item row="17" column="3">
    <widget class="QLineEdit" name="lineEdit_10"/>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_1"/>
   </item>
   <item row="13" column="3">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="20" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLineEdit" name="lineEdit_8"/>
   </item>
   <item row="10" column="3">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="19" column="3">
    <widget class="QLineEdit" name="lineEdit_11"/>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogToInsert</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogToInsert</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
